fabric-networking-v1/src/main/java/net/fabricmc/fabric/api/networking/v1/event/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The events for the fabric networking API.
+ *
+ * <p>Packet channel callbacks are for the change of channel acceptance on the
+ * other side of connection.
+ */
+
+package net.fabricmc.fabric.api.networking.v1.event;
```

Comment:
Do we specify the package-info in the rest of the api or is this just exclusively to here?

Reply:
I do the package info for javadocs and potential annotations in the future.